README.md

```diff
@@ -13,26 +29,39 @@ $ make
 $ sudo make install
 ```
 
+### OS X specific steps
+
+Several packages are required to complete this build, including `Python` and `pkg-config`.  To install all the dependencies on OS X using `brew`:
+
+```
+brew install cmake
+brew install protobuf
+brew install grpc
+brew install python
+brew install pkg-config
+```
+
 ## Getting started
 
-To initialize the LightStep library in particular, either retain a reference to
-the LightStep `opentracing::Tracer` implementation and/or set the global
-`Tracer` like so:
+To initialize the LightStep library in particular, either retain a reference to the LightStep `opentracing::Tracer` implementation and/or set the global `Tracer` like so:
 
 ```cpp
+#include <opentracing/tracer.h>
 #include <lightstep/tracer.h>
-using namespace lightstep;
 
 int main() {
-  // Initialize the LightStep Tracer; see lightstep::LightStepTracerOptions for
-  // tuning, etc.
-  LightStepTracerOptions options;
-  options.access_token = "YourAccessToken";
-  auto tracer = MakeLightStepTracer(options);
+  lightstep::LightStepTracerOptions options;
+  options.component_name = "C++ app";
+  options.access_token = "<LightStep access token>";
+  auto tracer = lightstep::MakeLightStepTracer(std::move(options));
 
-  // Optionally set the opentracing global Tracer to the above.
   opentracing::Tracer::InitGlobal(tracer);
 
+  auto span = tracer->StartSpan("demo");
+  span->Finish();
+
+  tracer->Flush();
```

Comment:
`tracer->Close();` accomplishes the same thing, but is part of the OT API.

Reply:
I can't find a Flush method. Maybe it's in the unreleased master branch?

Reply:
There's no Flush method in the OT API, but [Close](https://github.com/opentracing/opentracing-cpp/blob/master/include/opentracing/tracer.h#L155) will flush the tracer.

Reply:
:+1: